sources/core/objects/Breakable.cpp

```diff
@@ -0,0 +1,40 @@
+#include "Breakable.h"
+#include "Mob.h"
+#include "Projectiles.h"
+
+Breakable::Breakable(quint32 id) : Structure(id)
+{
+   damage_points_ = 1;
+}
+
+void Breakable::AttackBy(IdPtr<Item> item)
+{   
+    if (!item.IsValid())
+    {
+        return;
+    }
+    if (IdPtr<Screwdriver> scr = item)
+    {
+        Structure::AttackBy(item);
+        return;
+    }
+    damage_points_ -= item->damage;
+    if (damage_points_ <= 0)
+    {
+        Break();
+    }
```

Comment:
There are on-hit sounds in SS13 when you hit a grille or a window, so they should be added.